### PR TITLE
Use TimeTravel from ui-components repo

### PR DESCRIPTION
**Pull request by fbarl**
Finalizes transition to reusable TimeTravel component by removing all the shared code from Scope.

### CODE REVIEW

File: client/app/scripts/components/time-travel-wrapper.js [hunk 1]
```diff
@@ -78,12 +78,10 @@ function mapStateToProps(state) {
     topologyViewMode: state.get('topologyViewMode'),
     currentTopology: state.get('currentTopology'),
     timestamp: state.get('pausedAt'),
-    // Used only to trigger recalculations on window resize.
-    viewportWidth: state.getIn(['viewport', 'width']),
```
Comment (foot) [inline client/app/scripts/components/time-travel-wrapper.js]: Don't need this anymore?

File: client/app/scripts/components/app.js [hunk 2]
```diff
@@ -85,7 +85,7 @@ class App extends React.Component {
   }
 
   componentWillUnmount() {
-    window.addEventListener('resize', this.handleResize);
+    window.removeEventListener('resize', this.handleResize);
```
Comment (foot) [inline client/app/scripts/components/app.js]: 👍

File: client/app/scripts/components/time-travel-wrapper.js (hunk 1 above)
Comment (fbarl) [reply client/app/scripts/components/time-travel-wrapper.js]: No, the TimeTravel in `ui-components` has its own resize listener.

**Comment by fbarl** [comment]
@jpellizzari I gave this PR to @foot for a quick review in hope that merging it would fix https://github.com/weaveworks/service-ui/issues/1286 (it didn't help, but the issue was fixed later).

**Pull request merged**